### PR TITLE
ZK-4763: Radiogroup selectedIndex error after radio moved

### PR DESCRIPTION
**Pull request by plijyfes**

### CODE REVIEW

File: zktest/src/archive/test2/config.properties [hunk 1]
```diff
@@ -2937,6 +2937,7 @@ B90-ZK-4431.zul=A,E,Multislider
 ##zats##B96-ZK-4804.zul=A,E,Tree,scroll,position
 ##zats##B96-ZK-4636.zul=A,E,Slider,drag,curpos
 ##manually##B96-ZK-4784.zul=A,E,Radio,Radiogroup,MemoryLeak,detach
+##zats##B95-ZK-4763.zul=A,E,Radio,Radiogroup
```
Comment (DevChu) [inline zktest/src/archive/test2/config.properties:2940]: 96

Comment (DevChu) [inline zktest/src/archive/test2/config.properties:2940]: 96

File: zktest/test/java/org/zkoss/zktest/zats/test2/B96_ZK_4763Test.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+Copyright (C) 2021 Potix Corporation. All Rights Reserved.
+*/
+package org.zkoss.zktest.zats.test2;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.zkoss.zktest.zats.WebDriverTestCase;
+import org.zkoss.zktest.zats.ztl.JQuery;
+
+public class B96_ZK_4763Test extends WebDriverTestCase {
```
Comment (DevChu) [inline zktest/test/java/org/zkoss/zktest/zats/test2/B96_ZK_4763Test.java:19]: ZATSCase

**Review by DevChu — CHANGES_REQUESTED** [review]

**Review by scribetw — APPROVED** [review]

**Review by DevChu — APPROVED** [review]

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**